internal/k8shandler/reconciler.go

```diff
@@ -77,12 +84,12 @@ func SecretReconcile(requestCluster *elasticsearchv1.Elasticsearch, requestClien
 	})
 
 	if retryErr != nil {
-		return kverrors.Wrap(retryErr, "failed to update status for cert redeploys",
+		return false, kverrors.Wrap(retryErr, "failed to update status for cert redeploys",
 			"cluster", requestCluster.Name,
 			"retries", nretries)
 	}
 
-	return nil
+	return false, nil
```

Comment:
can we actually flip this? it seems odd that we'd return `false` for a successful operation